src/qml/filters/audio_highpass/meta.qml

```diff
@@ -7,4 +7,33 @@ Metadata {
     name: qsTr("High Pass")
     mlt_service: 'ladspa.1890'
     qml: 'ui.qml'
+    keyframes {
+        allowAnimateIn: true
+        allowAnimateOut: true
+        simpleProperties: ['wetness']
```

Comment:
This needs to list all of the simple keyframes properties. Looks like I overlooked that in the low pass filter. I will fix that, but you need to fix it for these new PRs.

Reply:
Should be good to go now!